### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,*.pot  -L absolut,actualy,ans,ba,childs,currenty,dum,enew,ro,ser,yse`

### CODE REVIEW

**Comment by Harvie** [comment]
LOL. OK. Thanks

**Comment by Harvie** [comment]
can you please run the codespell again? there are some conflicts with previously merged PR :-(

**Comment by luzpaz** [comment]
Rebased

File: bCNC/CNC.py [hunk 1]
```diff
@@ -1252,7 +1252,7 @@ def compileLine(line, space=False):
             return None
 
         out = []  # output list of commands
-        braket = 0  # bracket count []
+        bracket = 0  # bracket count []
```
Comment (luzpaz) [inline bCNC/CNC.py:1255]: changed code here, fyi (see below as well)

**Comment by Harvie** [comment]
Thanks :-)

**Pull request merged**